python/util/templates.py

```diff
@@ -70,6 +70,7 @@ def generate_react_components(self):
         user = null,
         foundDomain = false,
         domain = null,
+        job = window.job,
```

Comment:
don't create a variable to just use it only once. it defeats the entire point of variables.

Reply:
does not appear changed